resources/assets/components/UserOverview/index.js

```diff
@@ -28,7 +28,9 @@ class UserOverview extends React.Component {
     this.api.get('api/v2/activity', {
       filter: {
         northstar_id: id,
-      }
+      },
+      orderByPosts: 'desc',
+      limit: 100,
```

Comment:
is 100 enough for now to expose all signups for a user? I don't think we have any users that have more than 100 signups...

Reply:
This made me really curious so I dug into it! It looks like there are 23 users in Thor with over 100 signups, with the most having 243.

Reply:
But one of them is Ben. And one of them used to work here. And 2 of them used to be an intern. And one of them is Diane. And, of course, one of them is Nami.

Reply:
And it's 32 not 23 whoops!

Reply:
oooo @katiecrane smart! I totally should have done that! Wondering if I should do a limit of `500`?

Reply:
I wonder if it would be a better approach to allow an `all` option for the `limit` param. So instead of using the default or trying to figure out the highest number we just return everything if `limit=all`

Reply:
I love this idea @sbsmith86 ! I just updated but it's a WIP bc I want to extract a little more. I'll let you know when it's ready for another 👀 !

Reply:
Or maybe because that parameter is expecting a number, if it is explicitly set to `null` we give all results?